uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +70,18 @@ protected function cleanup() {
     }
   }
 
+  private function loadRequiredData() {
+    $this->testContact = ContactFabricator::fabricate();
+    $requiredActivityTypes = ['Create User Account', 'Send Onboarding Email'];
+
+    civicrm_api3('OptionGroup', 'create', ['name' => 'activity_type']);
```

Comment:
**activity_type** is a core civicrm  option group so I think it is safe to assume it is already exist